modules/helfi_react_search/helfi_react_search.install

```diff
@@ -98,6 +98,12 @@ function helfi_react_search_update_9005() {
  * UHF-9613: Update events list field descriptions and translation.
  */
 function helfi_react_search_update_9006() : void {
+  $module_installer = \Drupal::service('module_installer');
+
+  if (!\Drupal::moduleHandler()->moduleExists('readonly_field_widget')) {
```

Comment:
We should add readonly_field_widget module to platform_config's composer.json file. This if condition should be reduntant after that.

Reply:
Ah, true. What was I thinking. 😅